lib/benchee/conversion/count.ex

```diff
@@ -111,16 +113,24 @@ defmodule Benchee.Conversion.Count do
   ## Examples
 
       iex> Benchee.Conversion.Count.best([23, 23_000, 34_000, 2_340_000])
-      :thousand
+      %Benchee.Conversion.Unit{long:      "Thousand",
+                               magnitude: 1000,
+                               short:     "K"}
```

Comment:
Great ideas... I thought about `name` myself but omitted it since it seemed to just duplicate the keys. But then again, it's one of those things where I don't mention it and if a reviewer has the same idea I'm sure it's a good one! :)

Also, yes I like label to short and using this in the doctests seems superb, why didn't I think of this myself? :+1: 

I think we can keep "long" around, with it being an upcoming option and us fixing up the tests there should not be much harm to keep it around